pkg/cmd/workspace_use.go

```diff
@@ -0,0 +1,28 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/hookdeck/hookdeck-cli/pkg/validators"
+)
+
+type workspaceUseCmd struct {
+	cmd *cobra.Command
+}
+
+func newWorkspaceUseCmd() *workspaceUseCmd {
+	lc := &workspaceUseCmd{}
+
+	lc.cmd = &cobra.Command{
+		Use:   "use",
+		Args:  validators.ExactArgs(1),
+		Short: "Select a default workspace for upcoming commands",
+		RunE:  lc.runWorkspaceUseCmd,
+	}
+
+	return lc
+}
+
+func (lc *workspaceUseCmd) runWorkspaceUseCmd(cmd *cobra.Command, args []string) error {	
+	return Config.UseProfile(args[0])
```

Comment:
Currently I'm not validating `args[0]` is a valid workspace yet. Probably should do that. But also I'm wondering if you think it makes sense for us to introduce a bit of interactivity like a select or something like that using [`promptui`](https://github.com/manifoldco/promptui) or something similar?